plugin/upstream_generator/parser/bq_parser.go

```diff
@@ -29,7 +24,7 @@ var (
 	specialCommentPattern     = regexp.MustCompile(`(\/\*\s*(@[a-zA-Z0-9_-]+)\s*\*\/)`)
 )
 
-func ParseTopLevelUpstreamsFromQuery(query string) []bigquery.ResourceURN {
+func ParseTopLevelUpstreamsFromQuery(query string) []string {
```

Comment:
why did we change it into `string`?

Reply:
same reason here https://github.com/goto/optimus/pull/157#discussion_r1357683688